server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -146,6 +154,9 @@ public StatusDTO status() {
         }
 
         CandlepinCapabilities caps = CandlepinCapabilities.getCapabilities();
+        if (keycloakEnabled) {
+            caps.add(CandlepinCapabilities.KEYCLOAK_AUTH_CAPBILITY);
```

Comment:
One odd thing here is that we're updating the capability only once the status endpoint is called. This will be fine for callers, but if we ever examine it internally, it won't work as expected until that endpoint is hit. Also, we're repeatedly adding it on every iteration, which is just churn on the hashset.

Is there a place in the Keycloak module proper that would be a better fit for this?